docs/satellite-clone.md

```diff
@@ -3,12 +3,9 @@
 #### Important Notes ####
 Before cloning:
   - It is recommended that the target server is network isolated. We take steps to disable communication from the target server to existing capsules and hosts, but the only way to ensure this is in an isolated environment.
-  - If your storage is slow (like NFS) and your pulp backup tar file is large (>100 gb), you might see memory errors while untaring pulp data during the playbook run. In this case don't include pulp_data.tar during the cloning process and instead copy over `/var/lib/pulp` from the source server to the target server after the cloning process. For example, from the source server:
-
-     ```console
-        # rsync -az /var/lib/pulp target_server:/var/lib/pulp
-    ```
-    Note that rsync command may take a while to complete to copy large data.
+  - If your storage is slow (like NFS) and your pulp backup tar file is large (>100 gb), you might see memory errors while untaring pulp data during the playbook run. In this case, don't include pulp_data.tar during the cloning process.  After the clone run, you have two options:
+    - To recover pulp content after the cloning process, follow these [steps](#recover-pulp-content-after-cloning).
```

Comment:
I think we should make a clear distinction that this is an option only when you clone without pulp_data.tar